docs-source/content/faq/namespace-management.md

```diff
@@ -0,0 +1,167 @@
+* [Delete a namespace from an operator's domain namespace list](#deleting-a-kubernetes-namespace-from-an-operator)
+* [Delete and recreate a Kubernetes namespace that an operator manages](#recreating-a-previously-deleted-kubernetes-namespace).
+
+See more in [Common Mistakes and Solutions]({{<relref "/userguide/managing-operators/using-the-operator/using-helm.md#common-mistakes-and-solutions">}}).
+
+#### Checking the namespaces that an operator manages
+You can find out the list of the namespaces that an operator manages using the `helm get values` command.
+For example, the following command shows all values of the operator release `weblogic-operator`, and the `domainNamespaces" list contains `default` and `ns1`. 
+.
+```
+$ helm get values weblogic-operator
+domainNamespaces:
+- default
+- ns1
+elasticSearchHost: elasticsearch.default.svc.cluster.local
+elasticSearchPort: 9200
+elkIntegrationEnabled: false
+externalDebugHttpPort: 30999
+externalRestEnabled: false
+externalRestHttpsPort: 31001
+image: oracle/weblogic-kubernetes-operator:2.3.1
+imagePullPolicy: IfNotPresent
+internalDebugHttpPort: 30999
+istioEnabled: false
+javaLoggingLevel: INFO
+logStashImage: logstash:6.6.0
+remoteDebugNodePortEnabled: false
+serviceAccount: default
+suspendOnDebugStartup: false
+
+```
+
+If you don't know the release name of the operator, you can use `helm ls` to list all releases.
+
+#### Adding a Kubernetes namespace to an operator
+If you want a WebLogic operator deployment to manage a namespace, you need to add the namespace to the operator's `domainNamespaces` list. Note that the namespace has to be pre-created, say using the `kubectl create` command.
+
+By adding a namespace to the `domainNamespaces` list, it tells the operator deployment or runtime 
+to initialize the necessary Kubernetes resources for the namespace so that the operator is ready to host WebLogic domain resources in that namespace.
+
+When the operator is running and managing the `default` namespace, the following example helm command adds namespace `ns1` to the `domainNamespaces` list, where `weblogic-operator` is the release name of the operator, and `kubernetes/charts/weblogic-operator` is the location of the operator's helm charts.
+
+```
+$ helm upgrade \
+  --reuse-values \
+  --set "domainNamespaces={default,ns1}" \
+  --wait \
+  --force \
+  weblogic-operator \
+  kubernetes/charts/weblogic-operator
+```
+
+{{% notice note %}}
+Changes to the `domainNamespaces` list may not be picked up by the operator right away because the operator 
+checks the changes to the setting periodically. The operator becomes ready to host domain resources in 
+a namespace only after the required `configmap` (namely `weblogic-domain-cm`) is initialized in the namespace.
+{{% /notice %}}
+ 
+You can verify if the operator is ready to host domain resources in a namespace by checking the existence of the required `configmap` resource.
```

Comment:
verify if the operator -> verify that the operator
by checking the existence  -> by confirming the existence